### PR TITLE
pacific: pybind/ceph_volume_client: Fix test_idempotency test failure

### PR DESCRIPTION
**Pull request by kotreshhr**
Fixes: https://tracker.ceph.com/issues/48830
Signed-off-by: Kotresh HR <khiremat@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins test make check

**Comment by kotreshhr** [comment]
The fix works and here is the teuthology run:

http://pulpito.front.sepia.ceph.com/khiremat-2021-01-24_07:35:02-fs-wip-khiremat-39038-fix-test_idempotency-distro-basic-smithi/

**Comment by kotreshhr** [comment]
jenkins test make check

File: src/vstart.sh [hunk 1]
```diff
@@ -659,6 +659,7 @@ EOF
         bluestore_spdk_mem = 2048"
         else
             BLUESTORE_OPTS="        bluestore block db path = $CEPH_DEV_DIR/osd\$id/block.db.file
+        bluestore block size = 10737418240
```
Comment (ajarr) [inline src/vstart.sh:662]: Is this needed?

**Review by ajarr — CHANGES_REQUESTED** [review]

File: src/vstart.sh (hunk 1 above)
Comment (kotreshhr) [reply src/vstart.sh:662]: My bad, got accidentally added to this PR. I will remove it.

**Comment by kotreshhr** [comment]
jenkins test make check

**Comment by kotreshhr** [comment]
jenkins test make check

**Comment by kotreshhr** [comment]
jenkins test make check

**Comment by kotreshhr** [comment]
jenkins test make check

**Review by vkmc — COMMENTED** [review]
Looks good to me. Thanks Kotresh!

**Review by ajarr — APPROVED** [review]

**Comment by kotreshhr** [comment]
Teuthology Run:

http://pulpito.front.sepia.ceph.com/khiremat-2021-02-08_16:32:10-fs-wip-khiremat-39038-idempotency-test-pacific-distro-basic-smithi/

**Pull request merged**